src/Stream/Stream_Api/HomeWork/StreamHW.java

```diff
@@ -2,130 +2,223 @@
 
 import sun.reflect.generics.reflectiveObjects.NotImplementedException;
 
-import java.util.IntSummaryStatistics;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
 public class StreamHW {
 
-    public static List<Integer> returnSquareRoot(List<Integer> numbers){
-        throw new NotImplementedException();
+
+    public static List<Integer> returnSquareRoot(List<Integer> num) {
+        List<Integer> numbers = Arrays.asList(1, 4, 16, 256);
+        return numbers
+                .stream()
+                .map(Math::sqrt)
+                .map(Double::intValue)
```

Comment:
try to avoid using duplicated function calls, maybe you can put it into one map?